modules/migration-upgrading-from-mtc-1-3.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/upgrading-migration-tool-3-4.adoc
+// * migration/migrating_4_1_4/upgrading-migration-tool-4-1-4.adoc
+// * migration/migrating_4_2_4/upgrading-migration-tool-4-2-4.adoc
+
+[id="migration-upgrading-from-mtc-1-3_{context}"]
+= Upgrading {mtc-short} 1.3
+
+If you are upgrading {mtc-full} ({mtc-short}) version 1.3.x, you must manually update the `indirectImageMigration` and `indirectVolumeMigration` parameters in the `MigPlan` custom resource (CR).
+
+Because the `indirectImageMigration` and `indirectVolumeMigration` parameters do not exist in version 1.3, their default value in version 1.4 is `false`, which means that direct image migration and direct volume migration are enabled. If the direct migration requirements are not fulfilled, the migration plan does not reach a `Ready` state.
+
+.Prerequisites
+
+* You must have upgraded {mtc-short} from version 1.3.x to 1.4.
+* You must have `cluster-admin` privileges.
+
+.Procedure
+
+. Log in to the target cluster.
+. Get the `MigPlan` CR:
++
+[source,terminal]
+----
+$ oc get migplan <migplan> -o yaml -n openshift-migration
+----
+
+. Change the following parameters to `true` and save the file:
++
+[source,yaml]
+----
+...
+spec:
+  indirectImageMigration: true
+  indirectVolumeMigration: true
+----
+
+. Apply the changes:
++
+[source,terminal]
+----
+$ oc replace -f <migplan>.yaml -n openshift-migration
+----
+
+. Verify the changes by viewing the updated `MigPlan` CR:
++
+[source,terminal]
+----
+$ oc get migplan <migplan> -o yaml -n openshift-migration
+----
```

Comment:
If you have output handy, I'd include the sample output.

Reply:
It's basically what is above:

spec:
  indirectVolumeMigration = true
  indirectImageMigration = true

The rest of the CR is unchanged.